cfme/common/vm.py

```diff
@@ -525,6 +526,17 @@ def unset_ownership(self):
             action=form_buttons.save)
         flash.assert_success_message('Ownership saved for selected {}'.format(self.VM_TYPE))
 
+    def open_console(self, console='Web Console', invokes_alert=False, cancel=False):
+        # TODO: implement vmrc vm console
+        if console in ['VM Console']:
+            raise NotImplementedError('VM Console not supported at this time')
+
+        navigate_to(self, 'Details')
+        # Click console button given by type
+        acc_btn(console, invokes_alert=invokes_alert)
```

Comment:
Some of the console types have an alert, Web Console does not so its False by default.